vendor/k8s.io/kubernetes/pkg/kubectl/cmd/get.go

```diff
@@ -299,6 +300,11 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 		if err := printer.PrintObj(obj, out); err != nil {
 			allErrs = append(allErrs, err)
 		}
+
+		// output empty list warning when no resources to display
+		if len(infos) == 0 {
+			fmt.Fprintf(os.Stderr, "%s\n", "There are no resources to display.")
```

Comment:
@fabianofranz do you think it would make more sense to output the current namespace / context instead of this? (There are two checkboxes on Trello that ask for more awareness of the current namespace / context and thought this might be a good place to at least partially address those: https://trello.com/c/04lpfTQR/451-8-cli-improved-flows-and-ux-in-the-cli-evg-ux-p3)